src/MyPlot/subcommand/TransferOwnershipSubCommand.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyPlot\subcommand;
+
+
+use MyPlot\MyPlot;
+use MyPlot\Plot;
+use pocketmine\command\CommandSender;
+use pocketmine\utils\TextFormat;
+
+class TransferOwnershipSubCommand extends SubCommand {
+
```

Comment:
coding inconsistency, should be:
```php
<?php
declare(strict_types=1);
namespace MyPlot\subcommand;

use imports;

class TransferOwnershipSubCommand extends SubCommand
{
//no eol
```

I notice that class definition formats aren't consistent throughout the project though, so @jasonwynn10 should probably choose one: `...{` or `...eol{`